.env.example

```diff
@@ -0,0 +1,4 @@
+DATABASE_URL=postgresql://user:password@localhost:5432/mydatabase
+POSTGRES_USER=user
+POSTGRES_PASSWORD=password
+POSTGRES_DB=mydatabase
```

Comment:
you already have user and password with db name in the database url. Either use full database url in your app or split in into five components and build them up on the app level to the full url